docs/Community/Governance.md

```diff
@@ -0,0 +1,161 @@
+Open Enclave SDK Community Governance Guidelines
+==========================
+
+**Note**
+This document reflects the proposed state of governance *after* migration to SIGs and WGs is completed.
+
+Like all documentation, this is a living document, a work in progress, representing
+approximately how the community operates and the principles which infuse our
+culture today.
+
+It should be edited and revised from time to time.
+
+Core Principles
+---------------
+
+The Open Enclave SDK community adheres to the principles of the Four Opens:
+
+* **Open Community**: We are welcoming and respectful of everyone.
+* **Open Design**: Project planning and design discussions happen in transparent, collaborative, and accessible ways.
+* **Open Development**: All ideas and contributions are welcome and accepted based on merit and alignment with project goals.
+* **Open Source**: All project code and dependencies are open source; see contribution guidelines below.
+
+Code of Conduct
+---------------
+
+The Open Enclave SDK community abides by the
+[OE SDK Code of Conduct](../CODE_OF_CONDUCT.md). Here is an
+excerpt:
+
+*In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to make participation in our project and our
+community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and
+expression, level of experience, education, socio-economic status, nationality,
+personal appearance, race, religion, or sexual identity and orientation.*
+
+As a member of the OE SDK project, you represent the project and your fellow
+contributors. We value our community tremendously and want to build and maintain
+a friendly, inclusive, collaborative environment for all our contributors and
+users.
+
+Community Governance Structure
+------------------------------
+
+The Open Enclave SDK community uses a
+[consensus-seeking](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making)
+approach to governance at all levels. To support a growing project, we have
+adopted a loose hierarchy wherein domain-specific decisions are handled within
+each domain, or Special Interest Group (**SIG**), cross-domain work is enabled
+through Working Groups (**WG**), and a Community Governance Committee (**CGC**)
+meets to review the overall structure, ensuring the health of the project.
+
+_NOTE: In lieu of a "technical steering committee", the Architecture SIG should
```

Comment:
I think this note can be removed from the governance section as you've also added similar  text in the technical discussions section below, which seems like a better fit for the topic. (maybe bump that last paragraph up to the front if it's not visible enough).